iogt/paginators.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import pagination
+
+
+class IoGTPagination(pagination.PageNumberPagination):
```

Comment:
it is generic just like paginator, other apps could also use it in the future (if necessary)

Reply:
I don't think this is good design because it means other apps would be relying on something in the `iogt` module which would prevent us from using the other apps in other projects, unless `iogt` is brought with them. There should be no dependency on `iogt` in the other apps.

I think we should move this paginator and the filter into `questionnaires`, and if they are needed somewhere else in future, we'll deal with that problem when we get to it. My preference would be for another module with common API code.

Reply:
done